src/components/UI/ButtonStyled/Button.styled.js

```diff
@@ -9,12 +9,13 @@ export const Button = styled.button`
 			align-items: center;
 			color: white;
 			font-weight: bold;
-			width: 100px;
+			width: 75px;
```

Comment:
Does your delete, edit and goodreads button have the exact same styling? Looks like they do. If yes, I'd recommend to create a default variant that you use for all three functionalities so that you don't repeat code.